### DISCUSSION TITLE
Simcom A7672E-FASE support

### DISCUSSION THREAD

**Discussion by MaxOvervoorde73** [waybyte/platform-logicrom#13] [Q&A]
Dear developer,

I was wondering if the A7672E-FASE is also supported in this repository.
If not, what do i have to configure to make it work?
I saw in the supported devices another A76XX series model. Also the chipset ASR1603 is supported with a different model.

I hope to hear from you soon!

Kind regards,
Max Overvoorde

**Comment by ajaybhargav** [comment]
Hi Max,

Do you have documentation and core firmware for A7672E-FASE? You can send them to my email address. We will try our best to integrate it to the SDK.

Regards,
Ajay Bhargav

**Comment by hgscania** [comment]
Hi. I have A7672E module but it's version is LASE and blt and gps not working. Anyone provide me FASE version? simcom support response tooooo late!

**Comment by ajaybhargav** [comment]
We can provide core firmware, but on module GNSS is not supported. Since there is no information on type of GNSS chip used and how it is connected to ASR chipset.

**Reply by hgscania** [reply]
I found part number "S2-109ZS-Z319S". What means z319s and which futures include in this part?
or what's meaning of each letter in part number?
